src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheck.java

```diff
@@ -204,21 +204,15 @@ public class RegexpCheck extends AbstractCheck {
      */
     private int duplicateLimit;
 
+    /** Tracks errors. */
+    private int errorCount;
```

Comment:
Please share a reason why we need this to be class field.
I thought we already agreed to move all to local variables

Reply:
it was because we had put logging into ```beginTree``` method. Now I have made it back to local variable